Client/Cliqz/Frontend/Browser/CliqzAppSettingsOptions.swift

```diff
@@ -128,6 +128,14 @@ class AdBlockerSetting: Setting {
 
 //Cliqz: Added Settings for redirecting to feedback page
 class SendCliqzFeedbackSetting: Setting {
+    
+    override init(title: NSAttributedString? = nil, delegate: SettingsDelegate?, enabled: Bool? = nil) {
+        super.init()
+        if delegate != nil{
```

Comment:
delegate is optional, so no need for a check. Direct assignment is enough and if it's nil nothing will be done and call the same initializer of super.